actioncable/lib/action_cable/channel/base.rb

```diff
@@ -230,7 +230,7 @@ def subscribe_to_channel
 
 
         def extract_action(data)
-          (data['action'].presence || :receive).to_sym
+          data['action'].presence.to_sym || :receive
```

Comment:
I think that in this case, we should add a test case to test `perform_action` with empty data. According to the code `:recieve` action should be dispatched, hence we should expect this in the test case. WDYT? Should I add the test case for that?